### ISSUE TITLE
Cannot deploy docker container

### ISSUE THREAD

**Issue by ghalpha** [ampache/ampache#3207]
### Describe the bug
Pulling the latest Docker image I am trying to deploy this and I keep getting a permission error. I have confirmed the directory owner is (www-data)33:33(www-data) and that the owner is also 33(www-data). I am going on about 8 hours of rebuilding and redeploying the container, creating volumes, creating shares, inspecting permissions. I have never had a docker give me this much of an issue before. So I am ready to farm out for some assistance. I am deploying this under a QNAP NAS, running Container Station/Portainer. I have searched what I think is everything and I can't seem to pinpoint the issue other than I know its a problem with permissions but the standard way of fixing this isn't working for me. 

### Screenshots
![image](https://user-images.githubusercontent.com/37425390/159600376-d515bd3c-a565-481a-9323-81ebcebbca18.png)

## Environment

* Ampache version:5.2.1
* Server operating system: QNAP OS

### Client type

* Web
* Ampache

**Comment by kuzi-moto** [comment]
Could you share your docker run command, or the docker-compose file used?

**Comment by ghalpha** [comment]
I am in the process of looking how to pull this specifically. The container is deployed via Portainer and its not painfully obvious how to export the run command or docker-compose.

**Comment by ghalpha** [comment]
Alright so after more troubleshooting today I have figured out the deployment issue on Docker on QNAP. It seems if you choose the "Advanced Folder Permissions" in the shared folders it breaks the permissions between the Docker host and the file share. If left disabled then everything works fine. It appears some way that QOS applies Linux based permissions, doesn't quite follow the correct implementation. So I finally narrowed it down to this setting, which is still confusing but that might make sense to some. 

I am going to close this issue with a comment and hopefully it helps someone else in the future. 

![image](https://user-images.githubusercontent.com/37425390/160260446-d669cd0c-0b2d-4c31-bb7a-692ae31d72f6.png)
